standard_rules/admin_assigned.yml

```diff
@@ -0,0 +1,119 @@
+AnalysisType: rule
+Filename: admin_assigned.py
+RuleID: Standard.AdminRoleAssigned
+DisplayName: Admin Role Assigned
+Enabled: true
+LogTypes:
+  - GCP.AuditLog
+  - GSuite.Reports
+  - OneLogin.Events
+Tags:
+  - DataModel
+Severity: Medium
+Reports:
+  MITRE ATT&CK:
+    - Privilege Escalation:Valid Accounts
+Description: Attaching an audit role manually could be a sign of privilege escalation
+Reference: TODO # TODO
```

Comment:
`Reference` is an optional field, if we don't have a good thing to reference we can drop it. Or perhaps we could link to the MITRE ATT&CK technique referenced above.